_posts/2019-01-05-its-time-to-use-spm.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 layout: epic
 title: It's time to use Swift Package Manager
-date: 2089-01-05
+date: 2019-01-05
```

Comment:
hah, this does break the URL though: 
https://twitter.com/teknik_tdr/status/1082399696373665793

but err, it would make it the top blog post for the next 70 years - so I think that's an OK trade now that I think about it

Reply:
Ha, yeah I hadn't thought about that. I've removed that commit from this PR – we can fix it later. Maybe we can hack Jekyll to keep the existing URL but with the corrected date?

Reply:
I'll break it, and re-announce the blog post on artsyopensource